tests/conf/i18n/translatehelpervisitor.js

```diff
@@ -0,0 +1,66 @@
+const TranslateHelperVisitor = require('../../../conf/i18n/translatehelpervisitor');
+const Translator = require('../../../conf/i18n/translator');
+const Handlebars = require('handlebars');
+const _ = require('lodash');
+
+async function createTranslator () {
+  const locale = 'fr';
+  return Translator.create(locale, [], { [locale]: { translation: {
+    'Hello': 'Bonjour',
+    'result': 'résultat',
+    'result_plural': 'résultats'
+  } } });
+}
+
+describe('TranslateHelperVisitor translates visited nodes', () => {
+  let translator;
+  /* global beforeAll */
```

Comment:
lets add this to the globals in the package.json (beforeEach is there as well)

Reply:
Oh, that's a much better spot!